libraries/core_libs/network/rpc/Test.cpp

```diff
@@ -281,7 +281,7 @@ Json::Value Test::get_all_nodes() {
   return res;
 }
 
-Json::Value Test::get_vote_weight(const Json::Value &param1) {
+Json::Value Test::get_coins_per_vote(const Json::Value &param1) {
```

Comment:
This should no be changed, it returns binomial distribution of weigh of vote per specific period and step

Reply:
It is just rename issue :)
https://github.com/Taraxa-project/taraxa-node/pull/1407#discussion_r787529065